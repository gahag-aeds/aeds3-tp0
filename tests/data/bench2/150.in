150 75
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 
Sub 14 63
Max 42 91
Add 93 142
Sum 90 139
Sub 2 51
Min 61 110
Sub 42 91
Sum 44 93
Add 91 140
Min 42 91
Add 87 136
Min 56 105
Add 19 68
Min 21 70
Sub 40 89
Sum 31 80
Sub 64 113
Max 1 50
Add 43 92
Sum 33 82
Add 13 62
Sum 44 93
Sub 24 73
Min 40 89
Sub 22 71
Max 76 125
Sub 42 91
Min 30 79
Add 85 134
Max 42 91
Sub 28 77
Sum 51 100
Add 35 84
Sum 72 121
Sub 92 141
Sum 36 85
Sub 32 81
Min 33 82
Add 79 128
Min 23 72
Sub 74 123
Sum 17 66
Sub 78 127
Min 44 93
Add 37 86
Sum 69 118
Sub 26 75
Min 1 50
Add 69 118
Sum 68 117
Add 33 82
Max 82 131
Sub 64 113
Max 8 57
Add 73 122
Min 37 86
Add 83 132
Sum 67 116
Sub 66 115
Min 19 68
Sub 8 57
Max 94 143
Add 69 118
Max 43 92
Add 65 114
Max 60 109
Sub 30 79
Min 97 146
Sub 44 93
Min 9 58
Add 71 120
Min 69 118
Add 25 74
Max 48 97
Sub 64 113
