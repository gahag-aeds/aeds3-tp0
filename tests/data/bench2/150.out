98
2668
2
2285
-2
1
-1
2126
93
2216
2312
-1
97
-3
97
2373
2949
2073
-4
-4
2005
-4
2862
-4
2914
97
92
-3
2963
-3
95
98
99
6
-6
-1
99
